source/renderer/app/components/widgets/forms/WalletsDropdownOption.js

```diff
@@ -4,30 +4,61 @@ import classnames from 'classnames';
 import { getColorFromRange } from '../../../utils/colors';
 import styles from './WalletsDropdownOption.scss';
 import StakePool from '../../../domains/StakePool';
+import LoadingSpinner from '../LoadingSpinner';
 
 export type WalletOption = {
   delegatedStakePool?: ?StakePool,
-  detail: string,
+  detail?: string,
   label: string,
   numberOfStakePools?: number,
   selected?: boolean,
+  syncing?: boolean,
```

Comment:
Please, rename this prop to `isSyncing`, for clarity

Reply:
Done